assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -214,6 +214,47 @@ const AddressForm = ( {
 					);
 				}
 
+				const customValidationHandler = (
+					inputObject: HTMLInputElement
+				) => {
+					if (
+						! isPostcode( {
+							postcode: values.postcode,
+							country: values.country,
+						} )
+					) {
+						inputObject.setCustomValidity(
+							__(
+								'Please provide a valid postcode',
```

Comment:
Take note of other notices in trunk: `Please enter a valid email address`. Can we update the wording to match?

Reply:
Absolutely. I just adjusted the wording to be in sync with the other messages.